### DISCUSSION TITLE
Key statistics required for reporting

### DISCUSSION THREAD

**Discussion by RedBeardEth** [BibliothecaDAO/economic-reporting#1] [Announcements]
## Key reporting  metrics

To efficiently balance the economy we require an ecosystem-wide view of influential key factors such as resource production and usage.

[Eve Online ](https://www.eveonline.com/news/view/monthly-economic-report-october-2022) does a great job of data collection and visualisation (public facing)

We should compile a list of relevant statistics to ensure we have the appropriate data collection pathways. To start:

Total free resources and lords supply
Production rate per resource
Total Food consumption
Food added to storehouse
Average Happiness
Building Uitilization
Total Vault Resources
Lords Claiming/goblin raiding
Armys built and Lost (both in resource values, and totals per unit)
Total resources Raided
AMM statistics (liquidity, total traded and price changes)

These should be collated by Orders and time

**Comment by ponderingdemocritus** [comment]
Yes, these are all great:

Other questions:
- What stack should we use for this
- IMO - we should go for bare-bones non-styled components - let's focus on data
- Could DUNE work?

**Comment by tenoxious** [comment]
One recommendation I got was Big Query as far as storage/usage. I'd imagine the reporting/visualizations would be internal.

**Comment by tenoxious** [comment]
@ponderingdemocritus are there "view" functions onchain that we can pull some of these from?
